test/integration/features/basic.feature

```diff
@@ -13,12 +13,14 @@ Checks whether CRC top-level commands behave correctly.
     And stdout should contain "Available Commands:"
     And stdout should contain "Flags:"
     And stdout should contain 
-    """Use "crc [command] --help" for more information about a command.
+    """
+    Use "crc [command] --help" for more information about a command.
     """
 
   Scenario: CRC setup
     When executing "crc setup" succeeds
-    Then stdout should contain "Starting Libvirt crc network"
+    Then stdout should contain "Caching oc binary"
+    And stdout should contain "Starting Libvirt crc network"
```

Comment:
This complete scenario will only going to work with linux, what we should do it to make our test platform independent from starting since in long run we will not struggle to patch it later. For me `crc setup` is a command and it can finish with an exist code. Adding stdout just to see the test string doesn't make much sense to me.

@anjannath @gbraad wdyt?